src/set_theory/game/basic.lean

```diff
@@ -64,94 +60,63 @@ by { rintro ⟨x⟩ ⟨y⟩ ⟨z⟩, apply pgame.le_trans }
 theorem le_antisymm : ∀ x y : game, x ≤ y → y ≤ x → x = y :=
 by { rintro ⟨x⟩ ⟨y⟩ h₁ h₂, apply quot.sound, exact ⟨h₁, h₂⟩ }
 
-/-- The relation `x < y` on games. -/
--- We don't yet make this into an instance, because it will conflict with the (incorrect) notion
--- of `<` provided by `partial_order` later.
-def lt : game → game → Prop :=
-quotient.lift₂ (λ x y, x < y) (λ x₁ y₁ x₂ y₂ hx hy, propext (lt_congr hx hy))
+/-- This instance is incompatible with that provided by `game_partial_order`, which is why it's made
```

Comment:
```suggestion
/-- This instance is incompatible with that provided by `game.game_partial_order`, which is why it's made
```
is the full name. Which is stupid.

Maybe this should say
```suggestion
/-- This instance is incompatible with that provided by `game.partial_order`, which is why it's made
```
and we should remove the weird name for that instance?

Reply:
I agree that it's a stupid name. I took the liberty to fix it here.